lib/retrieval-stats.js

```diff
@@ -227,15 +231,23 @@ export const recordCommitteeSizes = (committees, point) => {
   const nodeCounts = []
   /** @type {Array<number>} */
   const measurementCounts = []
-  for (const { subnets, participants, nodes, measurements } of stats.values()) {
+  /** @type {Array<number>} */
+  const majoritySizes = []
+  for (const { subnets, participants, nodes, measurements, majoritySize } of stats.values()) {
     subnetCounts.push(subnets.size)
     participantCounts.push(participants.size)
     nodeCounts.push(nodes.size)
     measurementCounts.push(measurements)
+
+    // Ignore committees that are too small or with no majority
+    if (majoritySize > 0 && measurements > 0) {
+      majoritySizes.push(Math.floor(100 * majoritySize / measurements))
```

Comment:
Why are we multiplying by 100? And dividing by measurements? Shouldn't `majoritySizes` contain `majoritySize`? I don't understand this translation

Reply:
Our histogram function requires the values to be integers. This code calculates the relative size of the majority vs. the committee in percentages.

How can I improve the code to make this intent more obvious?

Alternatively, I can modify the histogram function to support both integers and floats, but I am not convinced the extra complexity is worth it.

Reply:
Isn't `majoritySize` an integer though?

Reply:
Yes, it's an integer.

I am not interested in absolute values - a majority of 60 in a committee of 100 is not great, while a majority of 60 in the committee of 70 gives us a lot of confidence in the correctness of the reported result.

I am interested in the relative size of the majority vs. the entire committee. Is it a tight majority (60 out of 100) or a super majority (60 out ot 70)?

We have 1000 committees, it's not practical to record this relative size for every committee, that's why I am recording percentile values.

Reply:
Alright, can we change the name then please? We're not recording the majority size, we're recording it relative to something to the measurement count. I misread your intention from the variable names.

What about majorityToCommitteeSize?

Reply:
> Alright, can we change the name then please?

💯 

> We're not recording the majority size, we're recording it relative to something to the measurement count. I misread your intention from the variable names.
> 
> What about majorityToCommitteeSize?

We are not recording size, but a fraction expressed in percentages using the 0-100 range.

I guess that leads us to `majorityToCommitteeRatio`.

https://dictionary.cambridge.org/dictionary/english/ratio
> **the ratio of sth to sth** 
> The ratio of exports to imports also improved from 70.4% to 83.2%.